.github/workflows/qa.yml

```diff
@@ -0,0 +1,14 @@
+name: QA
+
+on: [ push, pull_request ]
+
+jobs:
+  spellcheck:
+    name: Spellcheck
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: cachix/install-nix-action@v19
+        with:
+          nix_path: nixpkgs=channel:nixos-22.11-small
+      - run: nix-shell -p typos --run "typos ."
```

Comment:
https://github.com/crate-ci/typos/blob/master/docs/github-action.md suggests using their own action, seems like it could be a little simpler than using the nix action.

Reply:
Oh, yes. The nix solution is complete overkill when there is typo action :D  - good catch!

Reply:
hmm, it seems like the typos action doesn't work.. :(

Reply:
Do we want this instability or use the nix approach instead? In nix, it is guaranteed to be stable 😋

Reply:
Ah there's a note in the doc: "for any of the examples above, make sure that you choose a release or commit as a version, and not a branch (which is a moving target)."

So probably instead of `uses: crate-ci/typos@master` it should be `uses: crate-ci/typos@v1.13.20`.